src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -139,6 +155,16 @@ void regclass_Core(py::module m) {
         py::arg("device_name"),
         py::arg("name"));
 
+    // todo: remove after Accuracy Checker migration to set/get_property API
+    cls.def(
+        "get_property",
+        [](ov::Core& self, const std::string& device_name, const std::string& name) -> py::object {
+            return Common::from_ov_any(self.get_property(device_name, name)).as<py::object>();
+        },
```

Comment:
Same comment as other `from_ov_any` and simple types. And possible solution:
```suggestion
        [](ov::Core& self, const std::string& device_name, py::object& name) {
            return Common::from_ov_any(self.get_property(device_name, PyAny(name)));
        },
```

Reply:
`py::object` -- removed

Reply:
done